velox/functions/prestosql/Map.cpp

```diff
@@ -40,8 +40,22 @@ class MapFunction : public exec::VectorFunction {
 
     static const char* kArrayLengthsMismatch =
         "Key and value arrays must be the same length";
-    static const char* kNullKey = "map key cannot be null";
-
+    auto checkNullsInKey =
+        [&](const auto& keysElements, auto offset, auto size) {
+          for (auto i = 0; i < size; ++i) {
+            if (keysElements->isNullAt(offset + i)) {
+              VELOX_USER_FAIL(kNullKeyErrorMessage);
+            }
+            if (keysElements->containsNullAt(offset + i)) {
```

Comment:
Why run this check twice?

Reply:
if the first fail the second is not reachable, 
do you suggest adding a new API containsNullAt that does not check the top level?